pkg/ebpf/tracer_windows.go

```diff
@@ -110,65 +108,36 @@ func (t *Tracer) expvarStats(exit <-chan struct{}) {
 				for metric, val := range stat.(map[string]int64) {
 					currVal := &expvar.Int{}
 					currVal.Set(val)
-					expvarEndpoints[name].Set(snakeToCapInitialCamel(metric), currVal)
+					if ep, ok := expvarEndpoints[name]; ok {
+						ep.Set(snakeToCapInitialCamel(metric), currVal)
+					}
 				}
 			}
 		}
 	}
 }
 
-// printStats can be used to debug the stats we pull from the driver
-func printStats(stats []network.ConnectionStats) {
-	for _, stat := range stats {
-		log.Infof("%v", stat)
-	}
-}
-
 // GetActiveConnections returns all active connections
 func (t *Tracer) GetActiveConnections(clientID string) (*network.Connections, error) {
-	t.bufferLock.Lock()
-	defer t.bufferLock.Unlock()
+	t.connLock.Lock()
+	defer t.connLock.Unlock()
 
-	activeConnStats, closedConnStats, bytesRead, err := t.driverInterface.GetConnectionStats(t.connStatsActive[:0], t.connStatsClosed[:0], t.driverBuffer)
+	activeConnStats, closedConnStats, err := t.driverInterface.GetConnectionStats()
```

Comment:
I am not a fan of the ownership model where the slices returned by driverInterface.GetConnectionStats()  need to be copied before the next call to GetConnectionStats(). Passing in a slice that gets filled in makes the ownership much clearer, and I would rather we keep that model if the performance is the same.  

The driverBuffer nicely abstracts the resizing logic -- could we pass those in instead of two `[]ConnectionStats` ?

Reply:
> I am not a fan of the ownership model where the slices returned by driverInterface.GetConnectionStats() need to be copied.

This was the existing model, because the returned slices pointed to the same underlying array the as the slices passed in. That is why I added this test for clobbering: https://github.com/DataDog/datadog-agent/blob/master/pkg/ebpf/tracer_test.go#L1797

I can move the buffers to the `Tracer`, but the returned slices will still need to be copied before the next call to `driverInterface.GetConnectionStats`. `GetActiveConnections` does so, by storing a connection in either the closed or active maps.

Reply:
PR updated.